tests/sheet/sheet_manipulation_plugin.test.ts

```diff
@@ -200,6 +202,49 @@ describe("Columns", () => {
       expect(model.getters.getColSize(sheetId, 1)).toBe(DEFAULT_CELL_WIDTH);
       expect(model.getters.getActiveSheet().numberOfCols).toBe(2);
     });
+
```

Comment:
all the tests should be moved with the other tests testing moving columns/rows, I don't think they belong here at all. And the test should be a bit more concise, I'm pretty sure you don't need 250 lines of tests to test a 3 lines fix.

Reply:
> all the tests should be moved with the other tests testing moving columns/rows,

There are only 2 test file with moveRow (or dispatch to move_colrow) not related to OT and I don't think the selection plugin test is a better place.

The test are in the suite "Rows/Cols > Correctly update references > On move", next to "Col/Row addition/deletion". It's an appropriate place

Reply:
The MOVE_COLUMNS_ROWS feature and your fix are in the selection plugin, why woudln't you want to test it in the selection plugin test file 😅 

The other tests in this file are related to core sheet manipulation add/remove columns). On the outside moveColumns sounds like the same, but it really isn't. It isn't a core command, it's just the selection UI plugin doing copy/paste things and make it look like a column moved.